src/ai/backend/manager/models/gql_models/image.py

```diff
@@ -806,6 +806,9 @@ async def mutate(
             if client_role != UserRole.SUPERADMIN:
                 if not image_row.is_customized_by(ctx.user["uuid"]):
                     raise GenericForbidden("Image is not owned by your account.")
+            for alias in image_row.aliases:
+                await session.delete(alias)
```

Comment:
If possible, it would be better to execute this as a single query using bulk delete instead of this code.

However, executing the following query results in the following error.

```py
            await session.execute(
                sa.delete(ImageAliasRow)
                .where(ImageAliasRow.image_id == image_id)
            )
```

```
2025-03-12 04:41:54.578 ERROR ai.backend.manager.api.admin [234212] ADMIN.GQL Exception: {'message': "UPDATE statement on table 'image_aliases' expected to update 1 row(s); 0 were matched.", 'locations': [{'line': 62, 'column': 3}], 'path': ['purge_image_by_id']}
```

Reply:
It seems to be an issue caused by inconsistency between the database and ORM because `ImageRow.get()` is called first, caching `ImageAliasRow` due to the `load_alias=True` option.

Reply:
Do you happen to know a good solution for this situation? @fregataa